evergreen/config.yml

```diff
@@ -1158,7 +1161,7 @@ buildvariants:
   - name: lint
     distros:
     - ubuntu2204-small
-  - name: core_sync_tests_group
```

Comment:
`core_sync_tests_group` does not seem to be used anymore

Reply:
Removed the task group